docs/sdk/src/guides/mod.rs

```diff
@@ -45,3 +45,4 @@ pub mod enable_metadata_hash;
 
 /// How to enable elastic scaling MVP on a parachain.
 pub mod enable_elastic_scaling_mvp;
+pub mod handling_parachain_forks;
```

Comment:
Some doc would be nice, to provide some info on this section:
```suggestion

/// Relay chain forks, and building on older relay parent.
pub mod handling_parachain_forks;
```